### DISCUSSION TITLE
Float point and marker labels at Slider go crazy

### DISCUSSION THREAD

**Discussion by rodvlopes** [quasarframework/quasar#15672] [General - Components / Directives / etc] [answered]
I'm having trouble dealing with 0.2 step and [min,max]: [0.4, 2] at Slider component 

<img width="598" alt="image" src="https://user-images.githubusercontent.com/95123/229598923-e73cedcf-22ca-4f92-bd34-0f26d11a3039.png">

Demo:
https://codepen.io/rodvlopes/pen/poOMOJQ?editors=101

Any idea to get this around?

**Comment by pdanpdan** [comment, marked as answer]
:) floating point in it's beauty
I think it's better to use integers (multiply and de-multiply in a computed)

Or custom format the marker labels

**Comment by rodvlopes** [comment]
Ok, working with floating points has its charm 😅

- Why did the floating point number break up with the imaginary number?
- Because they had real issues.

**I used the custom marker labels to solve this situation.**

Demo of the solution:
https://codepen.io/rodvlopes/pen/YzOmdvx?editors=1010

**Comment by pdanpdan** [comment]
Looks very nice
